### PR TITLE
feature: #52 Add Subject option

### PR DESCRIPTION
**Pull request by austininseoul**

### CODE REVIEW

File: frontend/lang/ca.js [hunk 1]
```diff
@@ -30,7 +30,10 @@ export default {
           },
           geography: {
             label: 'Geogràfia'
-          }
+          },
+          subject: {
+            label: 'Assignatura'
```
Comment (jmformenti) [inline frontend/lang/ca.js:35]: Change to 'Assumpte'

File: frontend/lang/es.js [hunk 2]
```diff
@@ -30,7 +30,10 @@ export default {
           },
           geography: {
             label: 'Geografía'
-          }
+          },
+          subject: {
+            label: 'Sujeta'
```
Comment (jmformenti) [inline frontend/lang/es.js:35]: Change to 'Asunto'

File: frontend/layouts/default.vue [hunk 3]
```diff
@@ -166,7 +171,6 @@ export default {
     }
   },
   mounted () {
-    this.$wikibase.$oauth.autoLoginByCookie()
```
Comment (jmformenti) [inline frontend/layouts/default.vue:169]: why remove this line?

File: frontend/layouts/default.vue [hunk 4]
```diff
@@ -194,7 +198,6 @@ export default {
     logout () {
       this.$store.commit('auth/logout')
       this.$notification.success(this.$i18n.t('auth.logout.success'))
-      this.$cookies.remove('oauth')
```
Comment (jmformenti) [inline frontend/layouts/default.vue:197]: why remove this line?

File: frontend/layouts/default.vue (hunk 3 above)
Comment (austininseoul) [reply frontend/layouts/default.vue:169]: its from a older version so...its pushing this change

File: frontend/layouts/default.vue (hunk 4 above)
Comment (austininseoul) [reply frontend/layouts/default.vue:197]: its from a older version so...its pushing this change

**Review by jmformenti — APPROVED** [review]

**Pull request merged**